tests/test_browser.py

```diff
@@ -4427,7 +4425,8 @@ def test_webgl_array_of_structs_uniform(self):
   @requires_graphics_hardware
   def test_webgl_offscreen_canvas_in_proxied_pthread(self):
     for asyncify in [0, 1]:
-      cmd = ['-s', 'USE_PTHREADS', '-s', 'OFFSCREENCANVAS_SUPPORT', '-lGL', '-s', 'GL_DEBUG', '-s', 'PROXY_TO_PTHREAD']
+      cmd = ['-s', 'USE_PTHREADS', '-s', 'OFFSCREENCANVAS_SUPPORT', '-lGL', '-s', 'PROXY_TO_PTHREAD']
+      # cmd += ['-sGL_DEBUG']
```

Comment:
I'm less sure about the GL_DEBUG ones. I think we should keep at least one, if not more.

Reply:
(LIBRARY_DEBUG is mostly useful for us devs, but GL_DEBUG might help users)